### PR TITLE
Add distributions range sliders

### PR DESCRIPTION
**Pull request by rubenaprikyan**
- Add ability to query distributions by step range and step density
- Add unified trace visualizer settings

### CODE REVIEW

File: aim/web/ui/src/services/models/runs/util.ts [hunk 1]
```diff
@@ -153,6 +153,10 @@ export function getMenuData(traceType: TraceType, traces: TraceRawDataItem[]) {
   const VisualizationMenuTitles = {
     images: 'Images',
     distributions: 'Distributions',
+    audios: 'Audios',
+    videos: 'Videos',
+    texts: 'Texts',
+    plots: 'Plotlies',
```
Comment (roubkar) [inline aim/web/ui/src/services/models/runs/util.ts:159]: let's change this to 'Plotly'

File: pytest.ini [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = --pdbcls=IPython.terminal.debugger:Pdb
```
Comment (roubkar) [inline pytest.ini:2]: should this file be included?

**Review by VkoHov — APPROVED** [review]

File: aim/web/ui/src/services/models/runs/types.d.ts [hunk 3]
```diff
@@ -12,14 +12,22 @@ export type IRunTraceModel = {
...
 };
 
-export type TraceType = 'distributions' | 'images';
+export type TraceType =
+  | 'distributions'
+  | 'images'
+  | 'audios'
+  | 'texts'
+  | 'videos'
+  | 'plots';
```
Comment (roubkar) [inline aim/web/ui/src/services/models/runs/types.d.ts:30]: is this 'plots' or maybe it should be 'plotly'

**Review by roubkar — APPROVED** [review]
Left a few comments for minor things, the rest looks good!

File: pytest.ini (hunk 2 above)
Comment (rubenaprikyan) [reply pytest.ini:2]: Changed accidentally, couldn't revert.

**Comment by rubenaprikyan** [comment]
Resolves #1114

**Pull request merged**